static/extended/gke.yaml

```diff
@@ -0,0 +1,34 @@
+- version: 1.33
+  extended: false
+- version: 1.32
```

Comment:
these versions should be strings not floats as well (can see it very clearly with `version: 1.3` instead of `version: 1.30` below